docs/guides/developer/docs/installation/source-linux.md

```diff
@@ -0,0 +1,90 @@
+Developer installation guide
+===========================
+
+These instructions outline how to install Opencast on a Linux system.
+
+
+1.Get Opencast Source
+--------------------
+
+You can get the Opencast source code by cloning the Git
+repository.
+
+Cloning the Git repository:
+
+    git clone https://github.com/opencast/opencast.git
+    cd opencast
+    git checkout develop
+
+2.Install Dependencies
+--------------------
+
+Please make sure to install the following dependencies.
+
+Required:
+
+    java-1.8.0-openjdk-devel.x86_64 / openjdk-8-jdk
+    ffmpeg >= 3.2.4
+    maven >= 3.1
+    unzip
+    gcc-c++
+    tar
+    bzip2
+
+Required as a service for running Opencast:
+
+    ActiveMQ >= 5.10 (older versions untested)
+
+Required for some services. Some tests may be skipped and some features
+may not be usable if they are not installed. Hence, it's generally a good idea to
+install them.
+
+    tesseract >= 3
+    hunspell >= 1.2.8
+    sox >= 14.4
+    synfig
+
+### Dependency Download
+
+Pre-built versions of most dependencies that are not in the repositories can be downloaded from the respective project
+website:
+
+* [Get FFmpeg](http://ffmpeg.org/download.html)
+* [Get Apache Maven](https://maven.apache.org/download.cgi)
+* [Get Apache ActiveMQ](http://activemq.apache.org/download.html)
+
+3.ActiveMQ Configuration
+--------------------
+
+Opencast comes with a basic configuration for ActiveMQ. Please follow the first
+configuration step to copy the XML file. [Message Broker Configuration](https://docs.opencast.org/develop/admin/configuration/message-broker/).
```

Comment:
It likely makes sense to add a note here to make sure you're using the same admin documentation version.  Unfortunately there's no way to correctly link directly to it, but something like

> ...copy the XML file.  The Message Broker Configuration page for the appropriate branch of Opencast can be found in the [Administration Documentation](https://docs.opencast.org/).

Reply:
It should be totally fine to just link to develop, shouldn't it? This is a quick installation guide for developers after all and it explains how to get the develop version of Opencast. Other versions and branches are not discussed anyway.

Reply:
Eh, fair enough.  I guess my concern lies with new developers being tasked by their institution to develop a fix, and starting work on the wrong branch.  In the end we can't protect against everything, so I guess linking to develop is fine.